src/main/java/dev/ebrydeu/spring_boot_library/domain/dto/MessageDto.java

```diff
@@ -1,15 +1,26 @@
 package dev.ebrydeu.spring_boot_library.domain.dto;
 
 import dev.ebrydeu.spring_boot_library.domain.entities.Message;
+import jakarta.validation.constraints.NotBlank;
+import jakarta.validation.constraints.NotNull;
+import jakarta.validation.constraints.PastOrPresent;
+import jakarta.validation.constraints.Size;
 
 import java.time.Instant;
 
 public record MessageDto(
         Long id,
+        @NotBlank(message = "Title must not be blank")
+        @Size(max = 100, message = "Title must not exceed 100 characters")
         String title,
+        @NotBlank(message = "Body must not be blank")
         String body,
+        @NotBlank(message = "Author must not be blank")
         String author,
```

Comment:
Should not we remove Author column? I think author is a name of the column that will be shown on web, we would need to map our user by id to it

Reply:
Yes we can probably remove it :)